### PR TITLE
[Database] TAPP-111: Remove pay1 and pay2 from Assignments and Offers

### PR DESCRIPTION
**Pull request by michellemtchai**
Please refer to issue #111

### CODE REVIEW

**Comment by dingtaoliu** [comment]
There is a merge conflict with the schema file because I just merged in a pr that included migrations which changed the schema version.  It seems like we will have this problem every time there are parallel branches that modify the schema, should we just remove the schema file from version control?

**Comment by michellemtchai** [comment]
I think we should if `schema.rb` can generated by rails instead of just be append to by it. Also, I noticed that when you switch from a branch that has deleted table a to a branch that still contain table a, table a doesn't come back on the second branch. This may become a bigger issue later on. Should I make another issue for that?

**Comment by michellemtchai** [comment]
I've fixed the conflict.

**Comment by michellemtchai** [comment]
I've fixed the conflict.

**Comment by dingtaoliu** [comment]
For the issue that you brought up, the problem is that the migrations are out of sync between the two branches.  I think we can fix this without resetting the entire database by dropping the 'schema_migrations' table that keeps track of migrations that have been run.  After this, we can run db:migrate again to get the db to the state we want it to be in a particular branch.

File: api/db/schema.rb [hunk 1]
```diff
@@ -10,7 +10,11 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
+<<<<<<< HEAD
```
Comment (George334456) [inline api/db/schema.rb]: Is this supposed to be here?

**Review by George334456 — CHANGES_REQUESTED** [review]
I think you left something inside, unless I'm looking at the wrong thing.

File: api/db/schema.rb (hunk 1 above)
Comment (michellemtchai) [reply api/db/schema.rb]: No, it's not supposed to be here. Great catch!

**Comment by michellemtchai** [comment]
I've removed the leftover line you caught.

**Review by George334456 — APPROVED** [review]
Good stuff

**Pull request merged**